content/master/concepts/managed-resources.md

```diff
@@ -839,96 +839,143 @@ Providers may define their own custom `Conditions`.
-`Reason: Available` indicates the Provider created the managed resource and it's
-ready for use. 
+### Type: Ready
 
-```yaml {copy-lines="none"}
+Condition `Type: Ready` indicates if the external resource is ready to use.
+
+The Condition `Type: Ready` and `Status: True` indicates Provider created
+the external resource and notified Crossplane it's ready to use. Either the
+external API explicitly indicated the resource is ready or Crossplane appears to
+have successfully created the external resource.
+
+{{<hint "important" >}}
+Crossplane doesn't update `Status: True` during a resource update.
+{{< /hint >}}
+
+The Condition `Type: Ready` and `Status: False` indicates the external resource
+isn't available. 
+
+
+#### Available
+{{<hover label="available" line="4">}}Reason: Available{{</hover>}} indicates 
+the Provider created the managed resource and it's ready for use. 
+
+```yaml {copy-lines="none",label="available"}
 Conditions:
   Type:                  Ready
   Status:                True
   Reason:                Available
 ```
-### Creating
 
-`Reason: Creating` indicates the Provider is attempting to create the managed
-resource. 
+#### Creating
 
-```yaml {copy-lines="none"}
+{{<hover label="creating" line="4">}}Reason: Creating{{</hover>}} indicates the 
+Provider is attempting to create the managed resource. 
+
+```yaml {copy-lines="none",label="creating"}
 Conditions:
   Type:                  Ready
   Status:                False
   Reason:                Creating
 ```
 
-### Deleting
-`Reason: Deleting` indicates the Provider is attempting to delete the managed
-resource. 
+#### Deleting
+{{<hover label="deleting" line="4">}}Reason: Deleting{{</hover>}} indicates the 
+Provider is attempting to delete the managed resource. 
 
-```yaml {copy-lines="none"}
+```yaml {copy-lines="none",label="deleting"}
 Conditions:
   Type:                  Ready
   Status:                False
   Reason:                Deleting
 ```
 
-<!-- vale off -->
-### ReconcilePaused
-<!-- vale on -->
-`Reason: ReconcilePaused` indicates the managed resource has a [Pause](#paused)
-annotation 
+#### Unavailable
+{{<hover label="unavailable" line="4">}}Reason: Unavailable{{</hover>}}
+indicates Crossplane expects the managed resource to be available, but the 
+Provider reports the resource is unhealthy.
 
-```yaml {copy-lines="none"}
+```yaml {copy-lines="none",label="unavailable"}
 Conditions:
-  Type:                  Synced
+  Type:                  Ready
   Status:                False
-  Reason:                ReconcilePaused
+  Reason:                Unavailable
 ```
 
+### Type: Synced
+
+Condition `Type: Synced` indicates Crossplane has checked the status of the managed
+resource with the Provider.
+
+The condition `Type: Synced` and `Status: True` means Crossplane successfully
+communicated with the Provider on the status of the manged resource and synced
+the managed resource's desired state with the observed state of the external
+resource.  
```

Comment:
I'd suggest leaving out "Crossplane communicated with the provider". I think it's misleading.

There's really no communication between Crossplane and the provider here, everything is done entirely by the provider.

You could either just talk about what the provider did, without mentioning Crossplane, or simplify and talk about what Crossplane did. I'd prefer the former, but the latter also makes sense if I think of "Crossplane" as a whole system, rather than a specific component.

Edit: Or by "Provider" do you mean "external system"? The capital P in Provider makes me think you're talking about the component of Crossplane, not the external system,